jobs/elasticsearch/templates/bin/ctl

```diff
@@ -38,11 +38,13 @@ case $1 in
     ulimit -u 4096
     sysctl -q -w vm.max_map_count=262144
 
+    (
     exec chpst -u $RUNAS:$RUNAS \
        /var/vcap/packages/elasticsearch/bin/elasticsearch \
-      -p $PIDFILE \
       <%= p("elasticsearch.exec.options", []).join(' ') %> \
       >>$LOG_DIR/elasticsearch.stdout.log 2>>$LOG_DIR/elasticsearch.stderr.log
+    ) &
+    echo $! > $PIDFILE
```

Comment:
Is this change related to this issue?

Reply:
Yes, it is: by executing the process in a subshell we make sure that the PID file exists once monit looks up for it. Otherwise we could end up in the situation where monit did not find the PID file and report the process as `execution failed`, but the process would still be running anyway.

Thanks for making it optional, it's a good idea :)